locales/en-US/app.ftl

```diff
@@ -705,9 +705,15 @@ ServiceWorkerManager--hide-notice-button =
 ## This is the settings component that is used in Call Tree, Flame Graph and Stack
 ## Chart panels. It's used to switch between different views of the stack.
 
-StackSettings--implementation-all-stacks = All stacks
-StackSettings--implementation-javascript = JavaScript
-StackSettings--implementation-native = Native
+StackSettings--implementation-all-stacks2 = <radio></radio> All frames
```

Comment:
I'm not a big fan of using `<radio></radio>` like this, but this was the easiest to implement.

Reply:
Is there any way to not expose this to localization? For example, what happens if someone moves it at the end of one of the strings?

Reply:
This would work, just it would be on the other side. This can be interesting for RTL localizers.

The alternative is to move the label outside of the input, linking them with the id. But I believe that our styles don't work like this currently so we'd need to implement something new. Also we'd probably need to split the title from the label text so that the title applies on both the radio button and its label.

Reply:
I'm trying to change this, as I really don't like it much.